src/fake_useragent/utils.py

```diff
@@ -49,43 +43,10 @@ def load() -> list[BrowserUserAgentData]:
         list[BrowserUserAgentData]: The list of browser user agent data, following the
             `BrowserUserAgentData` schema.
     """
-    data = []
-    ret: Union[list[BrowserUserAgentData], None] = None
     try:
-        json_lines = (
-            ilr.files("fake_useragent.data").joinpath("browsers.jsonl").read_text()
+        return json.loads(
```

Comment:
I would add a comment here explaining why this is done and what it does. Something like

```python
# Convert JSONL to single big JSON list to speed up loading by ~2x.
```

Reply:
Please don't forget about this, as the current approach is cryptic and devs in the future might want to change this for the slower but more typical approach.

Reply:
Done, thanks for the reminder! BTW, I was just wondering why browser data is stored as JSONL instead of a single JSON file? I'm guessing it's probably to reduce the diff?

Reply:
Jsonl was in the past chosen because of the fact we used to fetch data via a scraper. Benefit of Jsonl is the fact these are isolated json objects. Hence when using web scraping, these objects can be appended much easier to an existing Jsonl file. Especially because the order doesn't matter. And you could later follow up and append more data if you want to. That are all the benefits of json lines. 

With a normal json file you normally read in the file as if it's one big object. Same for writing to such a normal json file.